_layouts/default.html

```diff
@@ -3,36 +3,49 @@
 
   {% include head.html %}
 
-  <body>
+  <body class="no-banner">
 
-    {% include header.html %}
+    <header>
+      <div class="container">
+        {% include regions/menu.html %}
+      </div>
+    </header>
 
     <div class="region gold-sponsor">
       <div class="container">
-        <div class="col-md-12">
-          {% include regions/sponsor_gold.html %}
+        <div class="row">
+          <div class="col-md-12">
+            {% include regions/sponsor_gold.html %}
+          </div>
         </div>
       </div>
     </div>
 
     <main class="page-main" aria-label="Content">
       <div class="region top-content">
-        <div class="container">
-          {% include regions/home_top_content.html %}
+        <div class="row">
+          <div class="container">
```

Comment:
Structure and classes should be in this order:
```
<div class="container">
  <div class="row">
    ...
  </div>
</div>
```

Reply:
Thanks, I will fix this :)